app/views/install/compose.phtml

```diff
@@ -434,21 +430,25 @@ services:
       - _APP_LOGGING_PROVIDER
       - _APP_LOGGING_CONFIG
 
-  appwrite-maintenance:
+  appwrite-worker-usage:
```

Comment:
Can we fix this diff? 

`appwrite-maintenance` was already part of the compose file.. It need not be modified to `appwrite-worker-usage`

`appwrite-worker-usage` can be added as a new item for a cleaner diff